src/reverseProxy.js

```diff
@@ -95,15 +96,14 @@ var RdHandler = {
         headers: request.headers
       },
       clientRequest.id);
-
+      
```

Comment:
whitespaces can be removed .... from other files as well...

Reply:
done

Reply:
can be resolved.